packages/venia-concept/src/components/App/app.js

```diff
@@ -69,17 +69,7 @@ const App = props => {
                 addToast(errorToastProps);
             }
         }
-    }, [errors, handleDismiss]);
-
-    if (renderError) {
```

Comment:
Because there was a `useEffect` hook below this condition we got an error which said that we should always call the same amount of hooks in the same order every render. A conditional breaks this requirement, so I moved it below all the hooks.